test/scripts/filters/meta_generator.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const should = require('chai').should(); // eslint-disable-line
```

Comment:
We were changed to use globally `chai.should` in [this commit](https://github.com/hexojs/hexo/pull/3293/commits/9f52ef1149be241a148f1cdfc7aab8ad88403589).
So, no need this line.

Reply:
Removed